core/src/main/java/com/yahoo/ycsb/StatusCode.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package com.yahoo.ycsb;
+
+/**
+ * Status codes returned by databases.
+ */
+public class StatusCode {
+
+  /**
+   * The operation completed without error.
+   */
+  public final static int OK = 0;
+
+  /**
+   * The operation failed, with a generic error.
+   */
+  public static final int ERROR = -1;
+
+  /**
+   * The operation failed: the requested record was not found.
+   */
+  public static final int NOT_FOUND = -3;
+}
```

Comment:
Some other error codes that could be useful here are:
* UNEXPECTED_RESULT - successful operation, but result was not what was expected (eg, wrong number of records, etc.)
* CLIENT_ERROR - YCSB had an error running an operation (ex. given tablename was null)
* UNIMPLEMENTED - used with operation is not implemented for the given module

I'm sure there are others that could be useful, but initially just creating this centralized class will allow for easy updating and canonical status codes.